src/pages/Dashboard/Employees/index.tsx

```diff
@@ -3,19 +3,47 @@ import { sendSignInLinkToEmail } from 'firebase/auth';
 import { useParams } from 'react-router-dom';
 
 import { auth } from '../../../firebase';
-import Table from '../../../features/Table';
+import Table, { TableEmployee } from '../../../features/Table';
 import Button from '../../../components/Button';
 import SearchBar from '../../../features/SearchBar';
 import PageHeader from '../../../features/PageHeader';
 import ListBanner from '../../../features/ListBanner';
-import AddForm, { IFormData } from './AddForm';
+import AddForm from './AddForm';
+import DeleteForm from './DeleteForm';
 import { ITab } from '../../../features/TabList';
 import { IconNames } from '../../../components/Icon';
 import * as constant from './constant';
 import { addEmployee, getEmployees } from '../../../api/apiLayer';
 import { IEmployeeData } from '../../../interfaces';
 import { useAuth } from '../../../hooks/useEmployee';
 
+export declare interface IFormData {
+  employeeName: string
+  employeeEmail: string
+  role: string
+};
+
+const initialEmployee = [
+  {
+    id: 241,
+    name: 'John',
+    email: 'johnd@gmail.com',
+    role: 'Admin',
+  },
+  {
+    id: 242,
+    name: 'Jack',
+    email: 'jackd@gmail.com',
+    role: 'Manager',
+  },
+];
```

Comment:
like I explained above , the table data was never retrieved from firebase . it was static hard coded and I only changed it  to dynamic and add two employee so I test the functionality of the delete button 

I strongly agree and totally understand  , the employee table should be retrieved from firebse and fixing all the api errors that we keep see on the console. However , I was working only creating the delete button feature and I did not work on another problems .So I just created several workarounds to test the delete button feature by using employee data that was  hard coded.
I can create a JIRA issue for this and work on it. 


this is how it was the data was hard coded.

const Table = (): JSX.Element => {
  return (
    <div className='rounded-sm overflow-hidden border border-gray-100 bg-white'>
      <table className="table-auto border-collapse w-full">
        <thead className="bg-gray-100">
          <Row data={['Employee name', 'Email', 'Role']} head/>
        </thead>
        <tbody>
          <Row data={['John Doe', 'j.doe@email.com', 'Admin']}/>
          <Row data={['John Doe', 'j.doe@email.com', 'Admin']}/>
          <Row data={['John Doe', 'j.doe@email.com', 'Admin']}/>
          <Row data={['John Doe', 'j.doe@email.com', 'Admin']}/>
          <Row data={['John Doe', 'j.doe@email.com', 'Admin']}/>
          <Row data={['John Doe', 'j.doe@email.com', 'Admin']}/>
        </tbody>
      </table>
    </div>

Reply:
The code was retrieve from the database. It was also saved inside the employees variable. You removed the method and replaced it with hard coded text. It is not a problem if it you are testing the application but you need to replace it back after when you push for production. The code was hard coded because at the time of the creation the backend was not finished and there was no information coming from the backend. The connection was forgotten.

Reply:
to be deleted and refactored later when able to retrieve data from backend